wafer/schedule/tests/test_admin.py

```diff
@@ -209,7 +209,7 @@ def _make_time_filter(self, time):
         if time:
             return SlotStartTimeFilter(None, {'start': time}, Slot, self.admin)
         else:
-            return SlotStarTimeFilter(None, {'start': None}, Slot, self.admin)
+            return SlotStartTimeFilter(None, {'start': None}, Slot, self.admin)
```

Comment:
Woops. :?

Reply:
Indeed.

That we haven't tripped over this previously suggests that we need to look at the tests here more carefully at some point.